Makefile

```diff
@@ -33,15 +33,36 @@ endif
 #########################################
 # Rules for local development scenarios #
 #########################################
+.PHONY: downlaod-kubeconfigs
+download-kubeconfigs:
+	@echo "enter project name"; \
+	read PROJECT; \
+	echo "enter seed name"; \
+	read SEED; \
+	echo "enter shoot name"; \
+	read SHOOT; \
+	./hack/local_setup.sh --SEED $$SEED --SHOOT $$SHOOT --PROJECT $$PROJECT;
+
+.PHONY: local-mcm-up
+local-mcm-up: download-kubeconfigs
+	$(MAKE) start;
+
```

Comment:
keep the rules `download-kubeconfigs` and `local-mcm-up` under a section like `#When K8s cluster is Gardener Shoot#`